dotnetv3/dynamodb/FromSQL/ListItemsExample/ListItemsExample/ListItems.cs

```diff
@@ -1,38 +1,42 @@
-﻿// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved. 
+﻿// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
 // SPDX - License - Identifier: Apache - 2.0
-// snippet-start:[dynamodb.dotnet35.ListItems]
-using System;
-using System.Configuration;
-using System.Text;
-using System.Threading.Tasks;
-using Amazon;
-using Amazon.DynamoDBv2;
-using Amazon.DynamoDBv2.Model;
 
 namespace ListItems
 {
-    public class ListItems
-    {
-        public static async Task<ScanResponse> GetItemsAsync(IAmazonDynamoDB client, string table)
-        {
-            var response = await client.ScanAsync(new ScanRequest
-            {
-                TableName = table
-            });
+    using System;
+    using System.Configuration;
+    using System.Text;
+    using System.Threading.Tasks;
+    using Amazon;
+    using Amazon.DynamoDBv2;
+    using Amazon.DynamoDBv2.Model;
 
-            return response;
-        }
+    // snippet-start:[dynamodb.dotnet35.ListItems]
 
-        static void Main()
+    /// <summary>
+    /// Lists the items in an Amazon DynamoDB table. The name of the table and
+    /// the Amazon Region are defined in the application configuration file.
```

Comment:
AWS Region
(for consistency, AWS rather than Amazon)

Reply:
fixed